ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/Action/Decision/Add.cshtml.cs

```diff
@@ -127,8 +132,13 @@ private void PopulateCreateDecisionDtoFromRequest()
 			var dtr_month = Request.Form["dtr-month-request-received"];
 			var dtr_year = Request.Form["dtr-year-request-received"];
 			var dtString = $"{dtr_day}-{dtr_month}-{dtr_year}";
-			var dateStarted = DateTimeHelper.TryParseExact(dtString, out DateTime parsedDate) ? parsedDate : (DateTime?)null;
-						
+			var isValidDate = DateTimeHelper.TryParseExact(dtString, out DateTime parsedDate);
+
+			if (dtString != "--" && !isValidDate)
+			{
+				throw new InvalidOperationException($"{dtString} is an invaid date");

```

Comment:
I think this shouldn't be an invalidoperationexception - https://learn.microsoft.com/en-us/dotnet/api/system.invalidoperationexception?view=net-7.0
"[InvalidOperationException](https://learn.microsoft.com/en-us/dotnet/api/system.invalidoperationexception?view=net-7.0) is used in cases when the failure to invoke a method is caused by reasons other than invalid arguments. Typically, it is thrown when the state of an object cannot support the method call."

Actually I can see we've used this before, so maybe one to revisit later & update globally (we ought to have our own application exceptions really)

Reply:
Will be discussed in the tech debt meeting